plugins/ruby-foreman-ansible/debian/control

```diff
@@ -8,6 +8,9 @@ Homepage: https://github.com/theforeman/foreman_ansible
 
 Package: ruby-foreman-ansible
 Architecture: all
-Depends: ${misc:Depends}, bundler, foreman (>= 1.12.0~rc1), ruby-foreman-deface (<<2.0.0)
+Depends: ${misc:Depends}, bundler, foreman (>= 1.12.0~rc1),
+  ruby-foreman-deface (<<2.0.0), ruby-foreman-tasks (>= 0.8.1),
+  ruby-foreman-tasks (<<0.9.0), ruby-dynflow (>= 0.8.14),
```

Comment:
AFAICT, foreman-tasks in the 1.12 repos is  0.7.x, so that'll be nightly and 1.12. I presume?